package.json

```diff
@@ -10,11 +10,16 @@
     "@types/node": "^16.11.25",
     "@types/react": "^17.0.39",
     "@types/react-dom": "^17.0.11",
+    "@types/react-redux": "^7.1.22",
```

Comment:
lets use ~ instead of ^ as we want to update only minor versions and not major everywhere